scripts/apply-remote-migrations.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+app=${1}
+
+cd ./tdrs-backend
+
+echo "Install dependencies..."
+sudo apt install -y gcc
+sudo apt install -y graphviz
```

Comment:
Do we still need graphviz?

Reply:
what is it used for? if django doesn't need it to start up, theoretically it shouldn't be necessary. i've removed those lines but haven't yet tested it

Reply:
Discussed in office hours last week but for posterity, it's necessary during the `make_erd` pipeline for releases but shouldn't be necessary here.